### DISCUSSION TITLE
Technical Onboarding Completed

### DISCUSSION THREAD

**Discussion by mitul-goswami** [microsoft/SATechnicalOnboarding#47] [Questions]
I have completed the technical onboarding and filled the completion form but didn't receive any response of acceptance or rejection.

**Comment by SalemTracey** [comment]
@mitul-goswami - I see you submitted your from on Sat July 22. The Operations Team works M-F. Please provide them a few days to work thru the backlog. Thanks.

**Comment by SalemTracey** [comment]
@mitul-goswami this is resolved.
